app/javascript/alchemy_admin/components/spinner.js

```diff
@@ -0,0 +1,30 @@
+import { AlchemyHTMLElement, html } from "./alchemy_html_element"
+
+class Spinner extends AlchemyHTMLElement {
+  properties = {
+    size: { default: "medium" }
+  }
+
+  render() {
+    return html`
+      <div class="spinner ${this.size}">
```

Comment:
We need to prefix the size class

```suggestion
      <div class="spinner spinner--${this.size}">
```